modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -97,6 +100,8 @@
 
     private ExperimentTagAccessor experimentTagAccessor;
 
+    private PrioritiesAccessor prioritiesAccessor;
```

Comment:
@bargenilesh Can you please add a TODO somewhere in this class stating that these member variables need to be declared `final`